docs/how-to-docs/anchore-scan.md

```diff
@@ -86,6 +86,55 @@ steps:
     - push
     - tag
 ```
+#### Adding a file within `WHITELIST`
+We could also add a centralised place to list all vulnerabilities that we would like to whitelist.
+
+We can achieve this by doing the following 4 steps
+
+
+1. You need to make a repo for this example we will call it `my-cve-exceptions`
+
+2. Add a file In the root directory called `list.txt`
+
+```
+#This is the centralise list for all vulnerabilities that we would like to whitelist.
+CVE-2008-4318
+CVE-2020-25613
+
+```
+
+
+3. Now we need to go back to the `drone.yml` and add a new step to the file, here is how it will look. Make sure it's above scan-image step.
+
+```YAML
+- name: cloning-repo
+  image: alpine/git
+  commands:
+  - git clone https://github.com/UKHomeOffice/my-cve-exceptions
+  when:
+   event:
+   - push
+   - tag
+```
+
+4. Now we need to replace environment variable `WHITELIST` within scan-image step with `WHITELIST_FILE` and replace the value with the location of the file in this example `my-cve-exceptions/list.txt`
```

Comment:
Currently, the exceptions list file in https://github.com/UKHomeOffice/my-cve-exceptions is still called `list.csv`. It needs to be renamed to `list.txt`